docker/Dockerfile.omp_offload

```diff
@@ -0,0 +1,110 @@
+##############################################################
+# This Dockerfile contains the additional NVIDIA compilers, 
+# libraries, and plugins to enable OpenMP offloading and NVIDIA GPU 
+# acceleration of Devito codes.
+#
+# TOFIX: Uncomment line "self.cflags += ['-Xopenmp-target', '-march=sm_37']" in devito/arch/compiler.py and change sm_37 for sm_XX where XX is the compute capability of your GPU
```

Comment:
This is fixed now no?

Reply:
That's true. Will remove in the next commit.